quattor/client/rpms.pan

```diff
@@ -29,6 +28,11 @@ include { 'rpms/package_default_versions' };
     if(! exists(spma)) {
         SELF[spma] = nlist();
     };
+    ncd = escape('ncm-ncd');
```

Comment:
But can pkg_add or pkg_repl be used without defined version? If I try that, I get an error:
``user-initiated error: no default package version defined for package: ncm_2dncd``